src/popup/router/components/TemplateRenderer.vue

```diff
@@ -0,0 +1,37 @@
+<script>
+const renderNodeContent = (createElement, node) => (!node.childNodes.length
+  ? node.textContent
+  : Array.from(node.childNodes)
+    .filter((n) => [Node.ELEMENT_NODE, Node.TEXT_NODE].includes(n.nodeType))
+    .map((n) => {
+      switch (n.tagName) {
+        case 'em':
+          return createElement('em', renderNodeContent(createElement, n));
+        case 'mark':
+          return createElement('mark', renderNodeContent(createElement, n));
+        case 'strong':
+          return createElement('strong', renderNodeContent(createElement, n));
+        case 'br':
+          return createElement('br');
+        case 'p':
+          return createElement('p', renderNodeContent(createElement, n));
```

Comment:
Let's add everything that is needed for now, not for every __potentinal__ use case.